python/cudf/cudf/tests/test_string.py

```diff
@@ -3423,3 +3423,44 @@ def test_str_join_lists(sr, sep, string_na_rep, sep_na_rep, expected):
         sep=sep, string_na_rep=string_na_rep, sep_na_rep=sep_na_rep
     )
     assert_eq(actual, expected)
+
+
+def test_str_find_multiple():
+    s = cudf.Series(["strings", "to", "search", "in"])
+    t = cudf.Series(["a", "s", "g", "i", "o", "r"])
+
+    expected = cudf.Series(
+        [
+            [-1, 0, 5, 3, -1, 2],
+            [-1, -1, -1, -1, 1, -1],
+            [2, 0, -1, -1, -1, 3],
+            [-1, -1, -1, 0, -1, -1],
+        ]
+    )
+
+    assert_eq(s.str.find_multiple(t).to_pandas(), expected.to_pandas())
```

Comment:
Why are we comparing the `to_pandas` outputs and not comparing two `cudf.Series` objects directly? (Same for `Index` below).
```suggestion
    assert_eq(s.str.find_multiple(t), expected)
```

Reply:
This is required because `s.str.find_multiple(t)` results in `ListDtype(int32)` and `expected` is of `ListDtype(int64)` and there is no clean way to type-cast right now for `ListColumn`

Reply:
Let's add a comment to explain that, for future readers.

Reply:
```suggestion
    # We convert to pandas because find_multiple returns ListDtype(int32) and expected is ListDtype(int64).
    # Currently there is no easy way to type-cast these to match.
    assert_eq(s.str.find_multiple(t).to_pandas(), expected.to_pandas())
```

Reply:
Alternatively, can we provide a `dtype` to the `expected` constructor that specifies `ListDtype(int32)`?

Reply:
> Alternatively, can we provide a `dtype` to the `expected` constructor that specifies `ListDtype(int32)`?

Nope, that silently fails right now